app/javascript/components/signup_form.jsx

```diff
@@ -0,0 +1,74 @@
+// eslint-disable-next-line no-use-before-define
+import React from "react";
+import $ from "jquery";
+
+function SignupForm () {
+  const [name, setName] = React.useState("");
+  const [sname, setSname] = React.useState("");
+  const [email, setEmail] = React.useState("");
+  const [password, setPassword] = React.useState("");
+  const [confirmpassword, setConfirm] = React.useState("");
+  const [errorsList, setErrors] = React.useState([]);
+
+  const SendForm = (e) => {
+    e.preventDefault();
+    $.ajax({
+      async: false,
+      type: "POST",
+      url: "/create",
+      data: {
+        first_name: name,
+        second_name: sname,
+        email: email,
+        password: password,
+        password_confirmation: confirmpassword
+      },
+
+      success: function (result) {
+        setErrors(result.errors);
+      },
+      error: function (result) {
+        alert("Please try again");
+      }
```

Comment:
@meganduffy In response to your message saying you couldn't see any alert. I'm not sure why the file you were looking at wasn't updated but here it is updated